pyiron/pyiron.xml

```diff
@@ -1,17 +1,17 @@
-<tool id="pyiron_meta" name="PyIron" version="0.2.5">
+<tool id="pyiron_meta" name="PyIron" version="0.2.17">
     <description>meta tools</description>
     <requirements>
-        <requirement type="package" version="0.2.5">pyiron</requirement>
-        <requirement type="package" version="2019.06.05">lammps</requirement>
+        <requirement type="package" version="0.2.17">pyiron</requirement>
+        <requirement type="package" version="2020.03.03">lammps</requirement>
```

Comment:
but now you also need to depent on pyiron-data isn't it?
Or is pyiron-data a dependency of pyiron?

Reply:
https://github.com/conda-forge/pyiron-feedstock/blob/master/recipe/meta.yaml#L54 - yes it is a dependency